catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -144,13 +145,22 @@ public UpdateRequest process(UpdateRequest input)
             }
         }
 
+        Attribute validationErrors = metacard.getAttribute(Validation.VALIDATION_ERRORS);
+        Attribute validationWarnings = metacard.getAttribute(Validation.VALIDATION_WARNINGS);
+
+        if (validationErrors != null) {
+            errors.addAll(validationErrors.getValues());
+        }
+
+        if (validationWarnings != null) {
+            warnings.addAll(validationWarnings.getValues());
+        }
+
```

Comment:
`errors` and `warnings` collect information in the for loop above. Pre-existing error & warnings will get added to these new lists.